### PR TITLE
MF-223: Fix error on End Visit action

### PR DESCRIPTION
**Pull request by ghost**
#MF-223
Fixed unexpected error on End visit action
Added null check fro dimensions as they can be empty
Specified jest-dom setup to be used in tests
Added unit tests for Visit Button component

### CODE REVIEW

File: src/widgets/visit/visit-button-component.tsx [hunk 1]
```diff
@@ -224,9 +226,8 @@ const hideModal = () => {
   newModalItem({ component: null, name: null, props: null });
 };
 
-const visitUpdate = () => {
-  let visitData;
-  getStartedVisit.subscribe(item => (visitData = item.visitData));
+const visitUpdate = (currentVisit: any) => {
```
Comment (FlorianRappl) [inline src/widgets/visit/visit-button-component.tsx]: Can we type this? What is `any`?

**Review by FlorianRappl — COMMENTED** [review]
Love it in general. Great tests and really well fixed. Just wanted to make sure we have great types here, too :).

Awesome job! :rocket:

File: src/widgets/visit/visit-button-component.tsx (hunk 1 above)
Comment (ghost) [reply src/widgets/visit/visit-button-component.tsx]: Yes Sure. Will update this.

**Review by FlorianRappl — APPROVED** [review]
LGTM!

**Pull request merged**